src/main.py

```
@@ -0,0 +1,54 @@
+import firebase_admin
+from firebase_admin import credentials
+from firebase_functions import https_fn, options
+import os
+
+from src.api.endpoints import generation, research, assistants
```

Ruff (0.12.2)

`src.api.endpoints.generation` imported but unused

Remove unused import

(F401)